bigchaindb/common/schema/transaction_v1.0.yaml

```diff
@@ -13,7 +13,9 @@ required:
 - version
 properties:
   id:
-    "$ref": "#/definitions/sha3_hexdigest"
+    anyOf:
+    - "$ref": "#/definitions/sha3_hexdigest"
+    - type: 'null'
```

Comment:
I guess you're using this JSON Schema file for validating non-final, under-construction transactions as well, because of the validation happening inside to_dict or from_dict? This is a bit sad. I guess the idea is to change that in the future, so that the JSON Schema files can be for checking final transactions _only_. (Maybe there could be other JSON Schema files for checking non-final, under-construction transactions?) Did you create an issue to do this in the future?

Reply:
> Did you create an issue to do this in the future?

#1897